src/articles-view/ArticlesLister.jsx

```diff
@@ -0,0 +1,13 @@
+import ArticleCard from "./ArticleCard"
+
+const ArticlesLister = ({articles}) => {
+    return (
+        <ul className='article-list'>
+            {articles.map((article) => {
+                return <ArticleCard article={article}/>
```

Comment:
this will need a unique key. this is currently giving errors in the browser console.